test/e2e/bindata/assets/00_kueue-operator.crd.yaml

```diff
@@ -42,6 +42,39 @@ spec:
               config:
                 description: config that is persisted to a config map
                 properties:
+                  fairSharing:
```

Comment:
are these pieces directly related to this PR? they don't seem so to me

Reply:
They are not. But I realize that I didn't have any sync for the test data so the CRD rotted a little bit.

I can do this as a separate PR if you want. I added the sync when I generate the crd as part of this work.

Reply:
I guess I should be better about commits then. Out of habit I always squash.

Reply:
Opened up https://github.com/openshift/kueue-operator/pull/34.